cmd/jaeger/internal/extension/jaegerstorage/config.go

```diff
@@ -9,12 +9,14 @@ import (
 
 	memoryCfg "github.com/jaegertracing/jaeger/pkg/memory/config"
 	badgerCfg "github.com/jaegertracing/jaeger/plugin/storage/badger"
+	grpcCfg "github.com/jaegertracing/jaeger/plugin/storage/grpc/config"
 )
 
 // Config has the configuration for jaeger-query,
 type Config struct {
 	Memory map[string]memoryCfg.Configuration   `mapstructure:"memory"`
 	Badger map[string]badgerCfg.NamespaceConfig `mapstructure:"badger"`
+	GRPC   map[string]grpcCfg.Configuration     `mapstructure:"grpc"`
```

Comment:
I don't think this is how we want to do the configuration in V2, but we can address it more holistically later -- #5229